install/scripts/README.md

```diff
@@ -19,7 +19,7 @@ Installing using the repositories is beneficial because the hipSYCL installation
 We provide the following packages in both versions:
 
 Base packages: 
-* `hipSYCL-base<-nightly>`
+* `hipSYCL-base<-nightly>` (contains: llvm 11.0.1, boost: 1.7x with cmake )
 * `hipSYCL-base-rocm<-nightly>`
 HipSYCL packages:
 * `hipSYCL-omp<-nightly>`
```

Comment:
Can you remind me why we haven have those combinations still? With the plugin architecture we should be able to install all backends independently.

Reply:
That is true, these are for two reason: In order to keep consistent with the earlier packaging system, and to save the user typing hipsycl twice. Internaly `hipSYCL-omp-cuda` is just  a meta-package depending on `hipSYCL-omp` and `hipSYCL-cuda`